Riot/ViewController/RoomsViewController.m

```diff
@@ -65,4 +74,124 @@ - (void)destroy
     [super destroy];
 }
 
+#pragma mark - UITableView delegate
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    if (recentsDataSource.recentsDataSourceMode == RecentsDataSourceModeRooms && indexPath.section == recentsDataSource.directorySection)
```

Comment:
recentsDataSource.recentsDataSourceMode == RecentsDataSourceModeRooms  is useless